src/main/java/tsunami/security/scanner/utilities/KubeJavaClientUtil.java

```diff
@@ -35,11 +33,18 @@
  */
 public final class KubeJavaClientUtil {
 
-  private static CoreV1Api coreV1Api = new CoreV1Api();
+  private static CoreV1Api coreV1Api;
```

Comment:
make this `private final` instead of `private static`. You will need to make all the methods non-static as well.